### PR TITLE
fix(ingest): patch lookml types and refactor ingestion sources layout

### PR DESCRIPTION
**Pull request by kevinhu**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: metadata-ingestion/setup.py [hunk 1]
```diff
@@ -199,28 +199,28 @@ def get_long_description():
...
         "kafka-connect = datahub.ingestion.source.kafka_connect:KafkaConnectSource",
         "ldap = datahub.ingestion.source.ldap:LDAPSource",
         "looker = datahub.ingestion.source.looker:LookerDashboardSource",
         "lookml = datahub.ingestion.source.lookml:LookMLSource",
         "mongodb = datahub.ingestion.source.mongodb:MongoDBSource",
-        "mssql = datahub.ingestion.source.mssql:SQLServerSource",
-        "mysql = datahub.ingestion.source.mysql:MySQLSource",
+        "mssql = datahub.ingestion.source.sql.mssql:SQLServerSource",
+        "mysql = datahub.ingestion.source.sql.mysql:MySQLSource",
         "oracle = datahub.ingestion.source.oracle:OracleSource",
```
Comment (hsheth2) [inline metadata-ingestion/setup.py:219]: seems like you missed hive, oracle, and the generic sqlalchemy source

File: metadata-ingestion/src/datahub/ingestion/source/lookml.py [hunk 2]
```diff
@@ -478,6 +483,8 @@ def _get_upstream_lineage(self, looker_view: LookerView) -> UpstreamLineage:
 
     def _get_field_type(self, native_type: str) -> SchemaFieldDataType:
         field_type_mapping = {
+            **POSTGRES_TYPES_MAP,
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/lookml.py:486]: would be good to move this up to the top-level of the file (only initialize the dict once + allows dynamic modification if needed)

File: metadata-ingestion/src/datahub/ingestion/source/lookml.py [hunk 3]
```diff
@@ -509,14 +516,21 @@ def _get_field_type(self, native_type: str) -> SchemaFieldDataType:
             "sum_distinct": NumberTypeClass,
         }
 
-        if native_type in field_type_mapping:
-            type_class = field_type_mapping[native_type]
-        else:
+        type_class = field_type_mapping.get(native_type)
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/lookml.py:519]: let's make sure this is comprehensive

**Review by hsheth2 — APPROVED** [review]
LGTM

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**